int64 robot_id
std_msgs/String cmd
std_msgs/String data